docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -253,6 +253,16 @@ Your extension project should be setup as a multi-module project with two submod
 
 TIP: You may want to use the `create-extension` mojo of `io.quarkus:quarkus-maven-plugin` to create these Maven modules - see the next section.
 
+==== Naming
+
+If your extension are not part of Quarkus it should have a distinct namespace. Do not start your extension group and artifact id with `quarkus`. This is to avoid name clashes between extensions.
```

Comment:
```suggestion
If your extension is not part of Quarkus it should have a distinct namespace. Do not start your extension group and artifact id with `quarkus`. This is to avoid name clashes between extensions.
```